### PR TITLE
SSD Example: Fixed training bug

### PR DESCRIPTION
**Pull request by Prasad9**
* Fixed import error bugs
* Fixed character encoding issue
* Converted Python 2 to Python 3

### CODE REVIEW

**Comment by szha** [comment]
@zhresho

File: example/ssd/symbol/common.py [hunk 1]
```diff
@@ -283,7 +283,7 @@ def multibox_layer(from_layers, num_classes, sizes=[.2, .95],
             step = (steps[k], steps[k])
         else:
             step = '(-1.0, -1.0)'
-        anchors = mx.symbol.contrib.MultiBoxPrior(from_layer, sizes=size_str, ratios=ratio_str,
+        anchors = mx.contrib.symbol.MultiBoxPrior(from_layer, sizes=size_str, ratios=ratio_str,
```
Comment (zhreshold) [inline example/ssd/symbol/common.py]: We should be able to use mx.symbol.contrib now.

**Comment by zhreshold** [comment]
Generally LGTM, but binary change looks weird

**Comment by Prasad9** [comment]
@zhreshold, the reason for that binary change is there is a strange character appearing in line number 205 of [train_net.py](https://github.com/apache/incubator-mxnet/blob/master/example/ssd/train/train_net.py) after the None. Because of that, TextEdit in Ubuntu is not able to open that file though other editors are able to do it but that character will appear. It will not be visible from browser. Also, if we run the code of [train.py](https://github.com/apache/incubator-mxnet/blob/master/example/ssd/train.py) without removing that character, we will get the following error:
```
Traceback (most recent call last):
  File "train.py", line 23, in <module>
    from train.train_net import train_net
ValueError: source code string cannot contain null bytes
```
However, let me know if you wish to remove that change. I will revert that change and push the code again.

**Comment by szha** [comment]
@Prasad9 saving it as binary file may cause other troubles. could you look for a way to fix the null byte issue instead?

**Comment by zhreshold** [comment]
Okay, I see what's happening there. 
@Prasad9 Keep the change, but could you revert those symbol.contrib -> contrib.symbol, these changes are pointless.
@szha I think it's removing a null byte that won't be visible in github browser. The file is ok.

**Comment by Prasad9** [comment]
@zhreshold , @szha I have reverted the symbol.contrib to contrib.symbol like how it was originally.

**Comment by piiswrong** [comment]
@zhreshold ready to merge?

**Comment by zhreshold** [comment]
@Prasad9 please trigger a CI test
@piiswrong I think it's ready once passed.

**Comment by zhreshold** [comment]
@piiswrong Can we merge now? Changes not related to CI check.

**Pull request merged**